api/v1alpha1/serviceintegration_types.go

```diff
@@ -70,7 +87,7 @@ type ServiceIntegrationSpec struct {
 	LogsUserConfig *logsuserconfig.LogsUserConfig `json:"logs,omitempty"`
 
 	// External AWS CloudWatch Metrics integration Logs configuration values
-	ExternalAWSCloudwatchMetricsUserConfig *externalawscloudwatchmetricsuserconfig.ExternalAwsCloudwatchMetricsUserConfig `json:"external_aws_cloudwatch_metrics,omitempty"`
+	ExternalAWSCloudwatchMetricsUserConfig *externalawscloudwatchmetricsuserconfig.ExternalAwsCloudwatchMetricsUserConfig `json:"externalAWSCloudwatchMetrics,omitempty"`
```

Comment:
This type never sent its [configuration](https://github.com/aiven/aiven-operator/blob/main/controllers/serviceintegration_controller.go#L170). So it doesn't break things.